website/content/en/status/report-2024-10-2024-12/srcmgr.adoc

```diff
@@ -0,0 +1,14 @@
+=== New srcmgr group
+
+Contact: srcmgr <srcmgr@FreeBSD.org>
+
+A new link:https://www.freebsd.org/administration/#t-srcmgr[source management team] has been appointed by core@ to handle management of the FreeBSD src tree, akin to portmgr@ and doceng@ for the ports and docs trees, respectively.
```

Comment:
s/team/group/ for consistency with all other instances in this report. (Alternatively, use "team" throughout including in the report title above.)